plugins/destination/mssql/client/spec.go

```diff
@@ -29,6 +34,21 @@ func (s *Spec) SetDefaults() {
 	if len(s.AuthMode) == 0 {
 		s.AuthMode = AuthModeMS
 	}
+
+	if s.BatchSize == 0 {
+		const defaultBatchSize = 1000 // 1K
```

Comment:
I think the inline `const` approach is unnecessary, I would just set `s.BatchSize = 1000`

Reply:
yeah, it's for documenting purposes only